desktop/ui/src/main/java/org/datacleaner/widgets/tree/SchemaTree.java

```diff
@@ -670,4 +674,9 @@ public void filter(String searchTerm) {
             }
         }
     }
+
+    public void setIncludeLibraryNode(boolean includeLibraryNode) {
+        _includeLibraryNode = includeLibraryNode;
+    }
```

Comment:
Hmmm... I'm not entirely sure that this is the correct/best way when we're using Guice for injection. I think you're supposed to configure the injector, but honestly I've managed to avoid needing to know until now. :D

@kaspersorensen maybe we should get a crash course in Guice, and especially in the way we use it?